client/src/components/PizzaPlaceList.js

```diff
@@ -31,6 +32,7 @@ const PizzaPlaceList = (props) => {
         <div className='pizza-list'>
             <h1>Pizza Place List</h1>
             <ul>{pizzaPlaceReact}</ul>
+            <Link to='/pizza-places/new' >Add New Pizza Place</Link>
```

Comment:
inconsistent spacing. Remove the extra space just after your `to` prop
I also recommend adding a space around `Add New Pizza Place`